script.js

```diff
@@ -0,0 +1,95 @@
+let defaultResult = 0;
+
+let currentResult = defaultResult;
+
+let logEntries = [];
+
+function getUserInput() {
+  return parseInt(userInput.value);
+}
+
+// the description & result output of calculation
+
+function createOutput(numberBeforeCalc, operator, calcNumber) {
```

Comment:
can we name it as `prevResult` ?

Reply:
Yes @faran4engg  will commit changes soon.